src/rmkit/input/input.cpy

```diff
@@ -34,7 +34,8 @@ namespace input:
     clear():
       self.events.clear()
 
-    def marshal(T ev):
+    // marshal can update the event
+    def marshal(T &ev):
```

Comment:
I'm not seeing where marshal updates an event. Is this left over from a previous version, or am I missing something (likely since I don't know this code very well)?